core/src/main/scala/libretto/CoreLib.scala

```diff
@@ -546,7 +544,7 @@ class CoreLib[DSL <: CoreDSL](val dsl: DSL) { lib =>
             await
         }
 
-      implicit def junctionNeed: Junction.Negative[Need] =
+      given junctionNeed: Junction.Negative[Need] =
```

Comment:
Would be imho better without names.

But Libretto uses a lot of explicitly passed givens. I'm not sure this is the right way.

Passing givens explicitly should be mostly needed to distinguish incoherent given instances. Not to pass them like any other parameter.

As this is common here, this would need some rethinking of the implementation.

Work for later, but food for thought.

Reply:
I found one explicit use of this `given`, which can actually be changed to `summon[Junction.Negative[Need]]`.

As for `Need`, this would be the canonical (and only) instance of `Junction.Negative`, it's fine to make it anonymous.